zerver/context_processors.py

```diff
@@ -133,8 +133,10 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         'allow_search_engine_indexing': allow_search_engine_indexing,
     }
 
-    if realm is not None and realm.icon_source == realm.ICON_UPLOADED:
-        context['OPEN_GRAPH_IMAGE'] = '%s%s' % (realm_uri, realm_icon)
+    if realm is not None:
+        context['OPEN_GRAPH_URL'] = '%s%s' % (realm_uri, request.path)
+        if realm.icon_source == realm.ICON_UPLOADED:
+            context['OPEN_GRAPH_IMAGE'] = '%s%s' % (realm_uri, realm_icon)
```

Comment:
I think we need a default value for pages which don't have a realm set (e.g. the zulipchat.com root domain) ... actually probably this just means moving `context['OPEN_GRAPH_URL'] = '%s%s' % (realm_uri, request.path)` outside the conditional.

Reply:
Ah, thanks for the catch! I missed the fact that realm_uri was being set even when we don't have a realm set.  I fixed this, and added a test for this case.